mmv1/products/firebasehosting/Version.yaml

```diff
@@ -229,3 +236,34 @@ properties:
                 ```
               min_version: 'beta'
               required: true
+      - name: 'headers'
+        type: Array
+        description: |
+         An array of objects, where each object specifies a URL pattern that, if matched to the request URL path,
+         triggers Hosting to apply the specified custom response headers.
+        item_type:
+          description: |
+            A Header specifies a URL pattern that, if matched to the request URL path, triggers Hosting
+            to apply the specified custom response headers.
+          type: NestedObject
+          properties:
+            - name: 'glob'
+              type: String
+              description:
+                The user-supplied glob to match against the request URL path.
+              exactly_one_of:
+                - 'glob'
+                - 'regex'
+            - name: 'regex'
+              type: String
+              description:
+                The user-supplied RE2 regular expression to match against the
+                request URL path.
+              exactly_one_of:
+                - 'glob'
+                - 'regex'
```

Comment:
`exactly_one_of` doesn't work for the attributes inside the nested object of a list. It is a known issue. Can you please comment it out? https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of

Reply:
Okay.